lib/spack/docs/replace_conda_homebrew.rst

```diff
@@ -184,7 +184,7 @@ simply run the following commands:
 .. code-block:: console
 
    $ spack env activate myenv
-   $ spack concretize --force
+   $ spack concretize --force --fresh
```

Comment:
Can you add a couple of lines below on what `--fresh` does (similarly to what written for `--force`)?

Reply:
Good idea! I wasn't sure if `--fresh` was sufficiently covered elsewhere in the docs but I like think it's worth having a little block about it here too.